types/noble/index.d.ts

```diff
@@ -65,10 +66,10 @@ export declare class Peripheral extends events.EventEmitter {
 
 export interface Advertisement {
     localName: string;
-    serviceData: {
+    serviceData: [{
         uuid: string,
         data: Buffer
-    };
```

Comment:
Wrapping brackets around a type tells TypeScript that this is a tuple type; both its length and members are typesafe. Take a look at the playground as an example: https://www.typescriptlang.org/play/#src=declare%20class%20Buffer%20%7B%0D%0A%20%20%20%20static%20from(src%3A%20string)%3A%20Buffer%3B%0D%0A%7D%0D%0A%0D%0Atype%20ServiceData%20%3D%20%5B%7B%0D%0A%20%20uuid%3A%20string%2C%0D%0A%20%20data%3A%20Buffer%0D%0A%7D%5D%0D%0A%0D%0Aconst%20data%3A%20ServiceData%20%3D%20%5B%0D%0A%20%20%7B%20uuid%3A%20'foo'%2C%20data%3A%20Buffer.from('foo')%20%7D%2C%0D%0A%20%20%7B%20uuid%3A%20'bar'%2C%20data%3A%20Buffer.from('bar')%20%7D%0D%0A%5D

This probably should be annotated as `Array<{ uuid: string, data: Buffer }>` instead.

Reply:
Right you are @bioball ! Thanks. See my most recent commit.